src/Microsoft.IdentityModel.JsonWebTokens/JwtTokenUtilities.cs

```diff
@@ -431,6 +431,17 @@ private static long ParseTimeValue(JToken jToken, string claimName)
             throw LogHelper.LogExceptionMessage(new FormatException(LogHelper.FormatInvariant(LogMessages.IDX14300, LogHelper.MarkAsNonPII(claimName), jToken.ToString(), LogHelper.MarkAsNonPII(typeof(long)))));
         }
 
+        internal static string PrepareSecurityArtifact(string securityArtifact)
```

Comment:
>PrepareSecurityArtifact [](http://example.com/codeflow?start=31&length=23)

nit: In my opinion this method doesn't describe very well what its done, may I suggest something on the line of SanitizeForLog or RemovePII

Reply:
In addition it seems this method is better suited for the Microsoft.IdentityModel.Logging.LogHelper class.